airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py

```diff
@@ -164,3 +164,9 @@ def detailed_logger() -> Logger:
     logger.log_json_list = lambda l: logger.info(json.dumps(list(l), indent=1))
     logger.handlers = [fh]
     return logger
+
+
+@pytest.fixture(name="ignored_fields")
```

Comment:
Why do you need fixture for this? Fixture galore makes code vary hard to read, just use input .